charts/tidb-cluster/templates/discovery-deployment.yaml

```diff
@@ -45,4 +45,14 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-
+{{- if .Values.enableTLSCluster }}
+        volumeMounts:
+        - mountPath: /var/lib/tls
+          name: tls
+          readOnly: true
+      volumes:
+      - name: tls
+        secret:
+          defaultMode: 420
+          secretName: client-tls
+{{- end -}}
```

Comment:
IIRC, have you changed the way the certs was injected, by injecting certs in the program instead of mount certs into pod?

Reply:
Yes, it's been changed in #782